src/Illuminate/Database/Query/Builder.php

```diff
@@ -2300,6 +2300,18 @@ public function toSql()
         return $this->grammar->compileSelect($this);
     }
 
+    /**
+     * Get the Raw SQL representation of the query.
+     *
+     * @return string
+     */
+    public function toRawSql()
+    {
+        return array_reduce($this->getBindings(), function ($sql, $binding) {
+            return preg_replace('/\?/', is_numeric($binding) ? $binding : "'".$binding."'", $sql, 1);
```

Comment:
I'd suggest to add null check and return a 'null' sql string. It may be important in insert/update query.

![image](https://user-images.githubusercontent.com/379924/125885496-d58f5179-6d63-4bba-aea1-494151b8f24a.png)

Reply:
Also there are ```$this->grammar->quoteString()``` or ```$this->connection->quote()``` for quoting value.

Reply:
> I'd suggest to add null check and return a 'null' sql string. It may be important in insert/update query.
> 
> ![image](https://user-images.githubusercontent.com/379924/125885496-d58f5179-6d63-4bba-aea1-494151b8f24a.png)

Since insert/update statement return a boolean, it's actually no possible to run `toSql()` or `toRawSql()` on those statements.  Was there a particular way you were writing the statement where you had this scenario that might not work? Ran a few tests and  any `where('field', null)` was already converted by `toSql()` to `"field" is null`.